app/lib/job/backend.dart

```diff
@@ -49,7 +49,8 @@ class JobBackend {
         ],
       ).toString();
 
-  Future trigger(JobService service, String package, [String version]) async {
+  Future trigger(JobService service, String package,
```

Comment:
Please give this a single like documentation... that would make me very happy...

```
/// Triggers analysis/dartdoc for [package]/[version] if older than [updated]
```

if that's what it does...